src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/IntellisenseHelper.cs

```diff
@@ -355,14 +356,38 @@ public static void AddTopLevelSuggestionsForCursorType(IntellisenseData.Intellis
 
                     if (!intellisenseData.TryAddSuggestionForCurrentBinaryOp(checkForOptionSetOnly: true))
                     {
-                        if (info.ScopeIdentifier != default)
+                        var node = callNode.Args.Children[argPosition];
```

Comment:
>                      var node = callNode.Args.Children[argPosition]; [](http://example.com/codeflow?start=2&length=69)

does PAClient have code like this that we can copy instead? 
We should understand how PAClient is working

Reply:
Removed